generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go

```diff
@@ -13,42 +13,29 @@ import (
 )
 
 type {{cookiecutter.beat|capitalize}} struct {
-	config config.Config
-	done   chan struct{}
-	client publisher.Client
+	done       chan struct{}
```

Comment:
hm, with libbeat providing Done and maybe even closing client on shutdown, we still need a stop method?

This might even be a bigger change in API.

for example, if I need to unblock anything once done closes I can write:

```
go func(){
    <-b.Done
    bt.UnblockWhatever()
}()
```